src/views/Home/components/FAQ/index.scss

```diff
@@ -0,0 +1,75 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.faq-component {
+  margin-top: -1px;
+  background-size: cover;
+  background: linear-gradient(
+    var(--color-light-pink),
+    var(--color-lavender),
+    var(--color-sky-blue)
+  );
+  &__box {
+    margin: auto;
+    border: 7px solid rgba(255, 255, 255, 0.75);
+    border-radius: 59px;
+    padding: 10px;
+    display: table;
+    // height: 1400px;
+    width: 90%;
+    @include responsiveness.screen(large-tablet, phone) {
+      width: 50%;
+    }
+  }
+  &__faqtitle {
+    font-family: "Source Sans Pro", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    @include responsiveness.font($responsive: 4.1vw, $min: 35px, $max: 60px);
+    text-align: center;
+    color: var(--color-text-primary);
+  }
+  &__line {
+    width: 25%;
+    height: 11px;
+    background: rgba(255, 255, 255, 0.73);
+    margin: auto;
+    // @include responsiveness.screen(desktop, big-desktop) {
+    //   width: 20px;
+    // }
+  }
+  &__question {
+    font-family: Lato;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 24px;
+    line-height: auto;
+    color: #ffffff;
+    cursor: pointer;
+  }
+  &__answer {
+    font-family: Lato;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: auto;
+    color: #ffffff;
+    display: none;
+  }
+
+  &__questionsandanswers:after {
+    display: table;
+    clear: both;
+  }
+  &__column {
+    float: left;
```

Comment:
I looked into flex box but I couldn't get it to work as well since the items would always overflow past the box surrounding the component. Also we need to flex with a column which requires a set height for the container. When the items are expanded and closed the desired height changes.